test/test_other.py

```diff
@@ -16300,3 +16204,10 @@ def test_install(self):
     # Some files, such as as maintenance tools should not be part of the
     # install.
     self.assertNotExists('newdir/tools/maint/')
+
+  def test_small_js_flags(self):
+    self.btest('browser_test_hello_world.c', '0', cflags=['-O3', '--closure=1', '-sINCOMING_MODULE_JS_API=[]', '-sENVIRONMENT=web', '--output-eol=linux'])
```

Comment:
I don't think you can move this test to test_other.py like this.   The `best` method is designed to run a browser test.. its not supposed to work outside of the browser tests.   I'm not even sure how this runs at all here.. the test is build with `-sENVIRONMENT=web` which means it should really run on node at all.